### PR TITLE
Applied fixes from FlintCI

### PR DESCRIPTION
**Pull request by soullivaneuh**

### CODE REVIEW

File: src/Datagrid/ProxyQuery.php [hunk 1]
```diff
@@ -60,8 +60,6 @@ class ProxyQuery implements ProxyQueryInterface
 
     /**
      * PHPCR ODM Document Manager.
-     *
-     * @var DocumentManager;
```
Comment (dbu) [inline src/Datagrid/ProxyQuery.php:64]: should this really be removed?

Comment (OskarStark) [reply src/Datagrid/ProxyQuery.php:64]: Maybe it’s spotted because it ends with a semicolon?

Comment (dbu) [reply src/Datagrid/ProxyQuery.php:64]: ok the semicolon sounds wrong. might be https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/4913 - though here i think the fix would be to remove the semicolon

Comment (OskarStark) [reply src/Datagrid/ProxyQuery.php:64]: I fixed it in `2.x`, lets close this one, so we get a new fresh one from afterwards...

**Pull request closed without merging**